lesson__2/lesson_4.js

```diff
@@ -0,0 +1,7 @@
+const numbers = [-20, -10, 0, 10, 20, 30]
+
+function sortDesc(num) {
+  return num.sort(function(a, b) {
+    return a - b;
+  }).reverse()
+}
```

Comment:
Достаточно было бы написать:
```suggestion
function sortDesc(num) {
  return num.sort(function(a, b) {
    return b - a;
  })
}
```